src/store/modules/issue/getters.js

```diff
@@ -0,0 +1,11 @@
+export default {
+  sortBy: state => (index, descendant) => {
+    state.items.sort((a, b) => a[index].id - b[index].id)
```

Comment:
Do not modify state in getters, it's an anti pattern. Sort a copy of original array and then return it.